doc/tutorials/open_vswitch.rst

```diff
@@ -0,0 +1,163 @@
+#######################################
+This tutorial describes how to use Open Virtual Switch
+(OVS) bridge instead of ACRN bridge.
+
+.. note::
+   The OVS is integrated in ClearLinux ``service-os``
+   bundle, and the ClearLinux OS must be at least ``28900`` version.
+
+What is OVS
+***********
+Open vSwitch, abbreviated as OVS, is an open-source implementation of
+a distributed virtual multilayer switch which is used to provide a switching
+stack for hardware virtualization environments, while supporting multiple
+protocols and standards used in computer networks.
+
+Open vSwitch supports the following features:
+
+- Visibility into inter-VM communication via NetFlow,
+  sFlow(R), IPFIX, SPAN, RSPAN, and GRE-tunneled mirrors
+- LACP (IEEE 802.1AX-2008)
+- Standard 802.1Q VLAN model with trunking
+- Multicast snooping
+- IETF Auto-Attach SPBM and rudimentary required LLDP support
+- BFD and 802.1ag link monitoring
+- STP (IEEE 802.1D-1998) and RSTP (IEEE 802.1D-2004)
+- Fine-grained QoS control
+- Support for HFSC qdisc
+- Per VM interface traffic policing
+- NIC bonding with source-MAC load balancing, active backup, and L4 hashing
+- OpenFlow protocol support (including many extensions for virtualization)
+- IPv6 support
+- Multiple tunneling protocols (GRE, VXLAN, STT, and Geneve, with IPsec support)
+- Remote configuration protocol with C and Python bindings
+- Kernel and user-space forwarding engine options
+- Multi-table forwarding pipeline with flow-caching engine
+- Forwarding layer abstraction to ease porting to new software and hardware platforms
+
+For more detailed OVS information, please refer to `<https://www.openvswitch.org/>`_
+
+.. _enable_ovs_in_ACRN:
+
+How to enable OVS in ACRN
+*************************
+The OVS service is integrated in ClearLinux ``service-os`` bundle.
+After boot ACRN Service OS, run ``systemctl start openvswitch`` command to start OVS.
+And ``acrn-br0`` is a default bridge, which is created by ``systemd`` of SOS,
+it is used to support UOS network.
+
+.. figure:: images/default-acrn-network.png
+   :align: center
+
+   Default ACRN Network
+
+.. note::
+   If ``systemctl start openvswitch`` fails to start OVS service,
+   you may need to install ``openvswitch`` via ``swupd bundle-add network-basic`` command.
+
+How to use OVS bridge instead ACRN bridge
+*****************************************
+#. Disable acrn network configuration::
+
+   # cd /usr/lib/systemd/network/
+   # mv 50-acrn.network 50-acrn.network_bak
+
+#. Modify ``50-eth.network`` to enable DHCP on OVS bridge
+
+    .. code-block:: none
+
+       [Match]
+       Name=ovs-br0
+
+       [Network]
+       DHCP=ipv4
+
+#. Create OVS bridge::
+
+   # ovs-vsctl add-br ovs-br0
+
+#. Add ``ens4``, ``acrn_tap_LaaG`` into OVS bridge::
+
+   # ovs-vsctl add-port ovs-br0 ens4
+   # ovs-vsctl add-port ovs-br0 acrn_tap_LaaG
+
+#. Modify ``launch_uos.sh`` script to enable ``acrn_tap_LaaG`` device before launch UOS::
```

Comment:
Quick setup script still use ``tap0`` as the default ``virtio-net`` device name, so the ``sed`` command is also work for the ``launch_uos_<version>.sh`` script.

Reply:
OK, thanks.  Do we need to mention that the user might need to update the ``launch_uos_<version>.sh`` script here?

Reply:
Yes, I will add a ``note`` to inform user to replace launch uos script directory in the ``sed`` command in case they're using the quick setup script to setup the UOS.